.styleci.yml

```diff
@@ -0,0 +1,11 @@
+preset: symfony
+
+finder:
+    exclude:
+        - "spec"
+        - "tests"
+    path:
+        - "src"
```

Comment:
spec is there so that we can use the same config in all projects (without thinking about being tested with phpunit or phpspec)

Tests folder: you are right